rhel7/profiles/anssi_nt28_enhanced.profile

```diff
@@ -10,28 +9,112 @@ extends: anssi_nt28_intermediary
+    # R16 - Hardened package repositories
+    # When the distribution provides several types of repositories, preference
+    # should be given to those containing packages subject to additional
+    # hardening measures.
+    # Between two packages providing the same service, those subject to hardening
+    # (at compilation, installation, or default configuration) must be preferred.
+
+    # ==============================================
+    # R17 - Boot loader password
+    # A boot loader to protect the password boot must be to be privileged.
+    # This password must prevent any user from changing their configuration options.
     - grub2_password
     - grub2_uefi_password
 
-    # R25 Yama module sysctl configuration
+    # ==============================================
+    # R20 - Installation of secret or trusted elements
+    # All secret elements or those contributing to the authentication mechanisms
+    # must be set up as soon as the system is installed: account and administration
+    # passwords, root authority certificates, public keys, or certificates of the
+    # host (and their respective private key).
+
+    # ==============================================
+    # R24 - Disabling the loading of kernel modules
+    # The loading of the kernel modules can be blocked by the activation of the
+    # sysctl kernel.modules_disabledconf:
+    # Prohibition of loading modules (except those already loaded to this point)
+    # kernel.modules_disabled = 1
+
+    # ==============================================
+    # R25 - Yama module sysctl configuration
+    # It is recommended to load the Yama security module at startup (by example
+    # passing the security = yama argument to the kernel) and configure the
+    # sysctl kernel.yama.ptrace_scope to a value of at least 1.
     - sysctl_kernel_yama_ptrace_scope
 
-    # R29 User session timeout
+    # ==============================================
+    # R26 - Disabling unused user accounts
+    # Unused user accounts must be disabled at the system level.
+
+    # ==============================================
+    # R28 - Uniqueness and exclusivity of system service accounts
+    # Each service must have its own system account and be dedicated to it exclusively.
+
+    # ==============================================
+    # R29 - User session timeout
+    # Remote user sessions (shell access, graphical clients) must be closed
+    # after a certain period of inactivity.
     - accounts_tmout
     - sshd_set_idle_timeout
+    - sshd_idle_timeout_value=5_minutes
 
-    # R35 umask value
+    # ==============================================
+    # R35 - umask value
+    # The system umask must be set to 0027 (by default, any created file can
+    # only be read by the user and his group, and be editable only by his owner).
+    # The umask for users must be set to 0077 (any file created by a user is
+    # readable and editable only by him).
```

Comment:
which umask will actually be used in the rules?

Reply:
Good question, the umask value should be set by a variable. I'll check it out later.

Reply:
I made selection of user `umask` value explicit.